i18n.js

```diff
@@ -1,12 +1,20 @@
 import i18n from 'i18next';
 import { initReactI18next } from 'react-i18next';
-import localization from './pages/i18n/localization.json';
+import localizationJSON from './pages/i18n/localization.json';
+
+const { localization } = localizationJSON;
 
 i18n.use(initReactI18next).init({
+  lng: 'uk',
   resources: {
-    localization,
+    en: {
```

Comment:
why you return this code? "localization" object which includes all localization across all application, we need just import localization object and that's all, we need this because we have localizationGenerator which will generate json file with localization, when you write "en" in this file or "uk", its a hardcode, if in the future we will need a new language, we will definitely need to add it in this file as well